Scripts/whitesource-spdx.yml

```diff
@@ -75,11 +75,11 @@ jobs:
             done
             unset IFS
             echo === Generating SBOM ===
-            ws_sbom_generator -u ${{secrets.USERKEY}} -k ${{secrets.APIKEY}} -s $WS_PROJECTTOKEN -a $WS_URL -t tv -o ./whitesource-spdx
+            ws_sbom_generator -u ${{secrets.USERKEY}} -k ${{secrets.APIKEY}} -s $WS_PROJECTTOKEN -a $WS_URL -t tv -o ./Mend-spdx
 
-      - name: Publish WhiteSource SPDX Report
+      - name: Publish Mend SPDX Report
         uses: actions/upload-artifact@master
         with:
-           name: WhiteSource SPDX report
+           name: Mend SPDX report
```

Comment:
If you change the name of the file on line 78, it should be changed here as well